# 请求部分
int32 speed                #全局速度百分比 0-100
---
# 响应部分
bool success               # 操作是否成功
string message             # 返回消息或错误描述